src/hooks/useResponsiveSpacing.mdx

```diff
@@ -0,0 +1,147 @@
+import { Meta, Source } from "@storybook/blocks";
+import { Box } from "@chakra-ui/react";
+import useResponsiveSpacing from "../hooks/useResponsiveSpacing";
+
+<Meta title="Hooks/useResponsiveSpacing" />
+
+# useResponsiveSpacing
+
+| Hook Version | DS Version   |
+| ------------ | ------------ |
+| Added        | `Prerelease` |
+| Latest       | `Prerelease` |
+
+The `useResponsiveSpacing` hook returns responsive gap and padding values that
+align with NYPL design standards for responsive grids and columns. The values
+will adjust based on the width of the viewport.
+
+The `Template` component uses the `useResponsiveSpacing` hook for margin and
+grid spacing values.
+
+## Usage
+
+<Source
+  code={`
+import { 
+  useResponsiveSpacing,
+} from "@nypl/design-system-react-components";
+// ...
+const {
+  responsiveGap,
+  responsiveMargin,
+  responsivePadding,
+} = useResponsiveSpacing();
+`}
+  language="jsx"
+/>
+
+## responsiveGap
+
+The `responsiveGap` object is intended to be used as the gap values for
+repeating grids and table.
+
+- Mobile: `1rem` (16px)
+- Tablet: `1.5rem` (24px)
+- Desktop: `2rem` (32px)
+
+<Source
+  code={`
+const {
+  responsiveGap
+} = useResponsiveSpacing();
+// ...
+const gridStyles = {
+  display: "grid",
+  gap: responsiveGap,
+  gridTemplateColumns: "repeat(3, 1fr)",
+}
+<div style={ gridStyles }>
```

Comment:
This is a good example. Can you add a more Chakra-specific one? Such as `<Box display="grid" gap={responsiveGap}>...` to show it used as style props.